uint8 BLUE=0
uint8 GREEN=1
uint8 RED=2
uint8 led_type
uint16 timeout